src/Sprites/ArtReader.cpp

```diff
@@ -90,7 +90,7 @@ namespace ArtReader {
 		for (Animation animation : artFile.animations) {
 			actualFrameCount += animation.frames.size();
 
-			for (Frame frame : animation.frames) {
+			for (Animation::Frame frame : animation.frames) {
```

Comment:
This would be a good place to use `auto`.

Reply:
This loop and it's parent loop should both actually be const auto&. Thanks for pointing it out.